package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@axway/swagger-tools",
-  "version": "0.11.5",
+  "version": "0.12.0-0",
```

Comment:
Consider bumping to 1.0.0. Gives plenty of space if we need to major bump 0.11.5 for any reason while supporting v4